fsharp-backend/tests/httptestfiles/README.md

```diff
@@ -117,6 +124,10 @@ One of the automatic changes made by text editors is the trimming of whitespace
 at the end of lines of code. To handle this, any usages of `<SPACE>` within the
 request or response are replaced with a space character (` `).
 
+Additionally, any usages of `<IMPORT_DATA_FROM_FILE=name>` will result in the
+test runner replacing this string with the contents of the referenced binary
+file - a file with the corresponding name should exist in `httpbytestestfiles/data`.
```

Comment:
Should probably change that directory, given that it can be used by non-bytes test files.

Reply:
You're probably doing that in the followup commit I guess.